src/components/BackToTopButton/index.styles.tsx

```diff
@@ -0,0 +1,43 @@
+import styled from '../../helpers/style';
+
+export const BackToTopStyles = styled.div<{ open: boolean}>`
+  display: block;
+  z-index: 99;
+  position: fixed;
+  bottom: ${props => props.open ? '5px' : '-50px'};
+  right: 5px;
+  background: ${props => props.theme.colors['blue']};
+  padding: 20px 5px 10px 5px;
+  border-radius: 5px;
+  box-shadow: 0px 0px 1px 0px rgba(0, 0, 0, 0.6);
+  cursor: pointer;
+  transition: all ease-in-out 0.5s;
+
+  &.open {
```

Comment:
This section of the CSS should be removed now :)

Reply:
Thanks! I went thru it with fresh eyes and removed some other unnecessary css as well.